docs/development/testing_configuration_tests.rst

```diff
@@ -16,6 +16,11 @@ Installation
 
     pip install -r securedrop/requirements/develop-requirements.txt
 
+
+.. note:: The testinfra tests require support for libvirt VMs. To set up libvirt 
+  support, follow the instructions here: :ref:`libvirt_provider`.
```

Comment:
There's a bit of a proliferation of make targets/molecule commands going on. This could be a good choice for future cleanup. eg. have a make test, make test-infra, make-test-func, and make test-all, that are platform/provider agnostic, using the same kind of logic as make staging.